tests/testthat/test-dbSelect.R

```diff
@@ -0,0 +1,36 @@
+test_that("database has been defined and dbSelect function works", {
+
+  # this is a copy of the empty database.sqlite db which is in ./inst/testdata
+  db_name <- "skeleton.sqlite"
+
+  base_path <- app_sys("testdata")
+  
+  # 1. file exists?
+  testthat::expect_true(file.exists(file.path(base_path, db_name)))
+  
+  # 2. valid db? test this is a valid sqlite database
+  testthat::expect_equal(readLines(file.path(base_path, db_name), n =1, warn = FALSE), "SQLite format 3")  
+
+  # 3. can we connect?
+  testthat::expect_true(DBI::dbCanConnect(RSQLite::SQLite(), db_name))
```

Comment:
So I noticed that this when ran locally actually creates a skeleton.sqlite file in my testthat folder because you have `db_name` instead of `file.path(base_path, db_name)`. This is also interesting that no error is raised despite the SQLite database not existing.

Reply:
I am going to fix that